yt_dlp/extractor/sauceplus.py

```diff
@@ -0,0 +1,40 @@
+from .floatplane import FloatplaneBaseIE
+
+
+class SauceplusIE(FloatplaneBaseIE):
+    _VALID_URL = r'https?://(?:(?:www|beta)\.)?sauceplus\.com/post/(?P<id>\w+)'
+    _BASE_URL = 'https://www.sauceplus.com'
+    _HEADERS = {'Origin': _BASE_URL,
+                'Referer': _BASE_URL + '/'}
+    _TESTS = [{
```

Comment:
are `_HEADERS` used anywhere?
```suggestion
    _IMPERSONATE_TARGET = True
    _TESTS = [{
```